lib/internal/types/docker_types.go

```diff
@@ -62,6 +62,7 @@ type DockerImageConfig struct {
 	NetworkDisabled bool
 	MacAddress      string
 	OnBuild         []string
+	Labels          map[string]string
```

Comment:
Note to self: this is coming out of https://github.com/docker/docker/blob/master/api/types/container/config.go#L58.